### PR TITLE
set up nodeJS project and routers

### PR DESCRIPTION
**Pull request by blaz2f**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const bodyParser = require('body-parser');
```
Comment (sicsixdkn) [inline server.js:2]: body-parser middleware is included with Express >=4.16 so this is a little outdated

File: server.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+//let db = require("./dbConnection");
+
+app.use(bodyParser.urlencoded({ extended: false }));
```
Comment (sicsixdkn) [inline server.js:9]: Can use express.urlencoded() here instead

File: server.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+app.use(bodyParser.json());
```
Comment (sicsixdkn) [inline server.js:10]: Can use express.json() here instead

**Review by sicsixdkn — CHANGES_REQUESTED** [review]
Looks like a good setup, just a couple of small changes

File: server.js (hunk 1 above)
Comment (blaz2f) [reply server.js]: fixed!

File: server.js (hunk 2 above)
Comment (blaz2f) [reply server.js]: done

**Review by sicsixdkn — APPROVED** [review]
Looks good!

**Pull request merged**